File: solutions/1-beginner/1018/1018.cpp
#include <iostream>

int main() {
    int a[] = {100, 50, 20, 10, 5, 2, 1};
    int n; std::cin >> n;
    std::cout << n << '\n';
    for (int i = 0; i < 7; ++i) {
        std::cout << n / a[i] << " nota(s) de R$ " << a[i] << ",00\n";
        n %= a[i];
    }
    return 0;
}

File: solutions/1-beginner/1018/1018.js
let n = parseInt(require('fs').readFileSync(0, 'utf8'))

console.log(n)

for (let a of [100, 50, 20, 10, 5, 2, 1]) {
    console.log(`${Math.floor(n / a)} nota(s) de R$ ${a},00`)
    n %= a
}
